core/src/test/scala/org/apache/spark/SparkThrowableSuite.scala

```diff
@@ -321,4 +323,22 @@ class SparkThrowableSuite extends SparkFunSuite {
         |  }
         |}""".stripMargin)
   }
+
+  test("overwrite error classes") {
+    withTempDir { dir =>
+      val json = new File(dir, "errors.json")
+      FileUtils.writeStringToFile(json,
+        """
+          |{
+          |  "DIVIDE_BY_ZERO" : {
+          |    "message" : [
+          |      "abc"
+          |    ]
+          |  }
+          |}
+          |""".stripMargin)
+      val reader = new ErrorClassesJsonReader(Seq(errorJsonFilePath.toUri.toURL, json.toURL))
```

Comment:
Do you use the existing JSON file? How can you guarantee that one JSON overrides the standard one? Maybe you will add small one for testing to check that it overrides error-classes.json?

Reply:
This test uses 2 JSON files: the existing official one and a temporary one which was written in this test.